controllers/state_of_the_world.go

```diff
@@ -63,15 +71,122 @@ func NewPolicyMachineryController(manager ctrlruntime.Manager, client *dynamic.D
 }
 
 func buildReconciler(client *dynamic.DynamicClient) controller.ReconcileFunc {
+	authorinoCrSubscription := &controller.Subscription{
+		ReconcileFunc: NewAuthorinoCrReconciler(client).Reconcile,
+		Events: []controller.ResourceEventMatcher{
+			{Kind: ptr.To(kuadrantv1beta1.KuadrantKind), EventType: ptr.To(controller.CreateEvent)},
+			{Kind: ptr.To(kuadrantv1beta1.AuthorinoKind), EventType: ptr.To(controller.DeleteEvent)},
+		},
+	}
```

Comment:
Could all this be part of `NewAuthorinoReconciler()`?

Reply:
Not sure. Would need to play with it to see what that would look like. I will look into it.